### DISCUSSION TITLE
Error sodium in softaculous

### DISCUSSION THREAD

**Discussion by davide1295** [firefly-iii/firefly-iii#5940] [Firefly III]
Hello, I would like to install the "Firefly III" application via softaculous but I get the following error:

 The following errors occurred:
 Required PHP extension not found: sodium

 Can the error be resolved?
 Thank you.

**Comment by JC5** [comment]
Yes, by installing the required PHP extension sodium.

**Reply by davide1295** [reply]
The problem is that my hosting does not allow the installation of that extension, is it not possible to "bypass it"?

**Reply by JC5** [reply]
I'm afraid not.
